### PR TITLE
Regularly against next builds of React

### PR DESCRIPTION
**Pull request by yuxizhe**
https://github.com/necolas/react-native-web/issues/1484

Add regularly against next builds of React. Will test next version of react every 12 hours.

image

image

### CODE REVIEW

File: .github/workflows/tests_react_next.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '14.x'
+    - run: npm install -g yarn
+    - run: yarn install
+    - run: yarn upgrade react@next react-dom@next -W --dev
+    - run: yarn fmt:report
+    - run: yarn flow
+    - run: yarn lint:report
+    - run: yarn jest
```
[the remark this answers is not in the document]
Comment (yuxizhe) [reply .github/workflows/tests_react_next.yml:21]: OK.  by the way, shall we need to update `tests.yml`? `tests.yml` `s Jobs are seperate, we can use `yarn test` instead.
https://github.com/necolas/react-native-web/blob/09ec7b6e53920ca96485f87ef78531e0aaa04416/.github/workflows/tests.yml#L10-L53

[the remark this answers is not in the document]
Comment (necolas) [reply .github/workflows/tests_react_next.yml:21]: yeah

**Comment by necolas** [comment]
It's annoying that React 18 is failing the tests by printing a *warning* using `console.error`.

File: .github/workflows/tests_react_next.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (necolas) [reply .github/workflows/tests_react_next.yml]: oh sorry, actually let's keep the individual tests separate as it's easier to see what failed

[the remark this answers is not in the document]
Comment (yuxizhe) [reply .github/workflows/tests_react_next.yml]: OK

[the remark this answers is not in the document]
Comment (yuxizhe) [reply .github/workflows/tests_react_next.yml]: Done

**Comment by yuxizhe** [comment]
> It's annoying that React 18 is failing the tests by printing a _warning_ using `console.error`.

Most of us wont see those regularly tests. For example,  Nextjs regularly react@next tests broken about 2 years, its seems no on care 🥲 

https://github.com/vercel/next.js/actions/workflows/test_react_next.yml

**Comment by necolas** [comment]
Thanks for the PRs!

**Pull request closed without merging**